_includes/nav.html

```diff
@@ -20,6 +20,7 @@
     <ul class="navbar-nav ms-auto">
       <li class="nav-item"><a class="nav-link{% if page.url contains "/privacy-policy" %} active{% endif %}" href="{{site.url}}{{site.baseurl}}/privacy-policy"> Privacy Policy</a></li>
       <li class="nav-item"><a class="nav-link" href="https://github.com/voxpupuli"><i class="fa fa-github"></i> Github</a></li>
+      <li class="nav-item"><a class="nav-link" href="https://join.slack.com/t/voxpupuli/shared_invite/zt-2qps81elf-mp3bVZJZGqbZfKQTi4Q4bQ"><i class="fa fa-slack"></i> Slack</a></li>
```

Comment:
I just updated https://short.voxpupu.li/puppetcommunity_slack_signup

```
$ curl -I https://short.voxpupu.li/puppetcommunity_slack_signup
HTTP/2 302
location: https://join.slack.com/t/voxpupuli/shared_invite/zt-2qps81elf-mp3bVZJZGqbZfKQTi4Q4bQ
x-powered-by: Short.io/Edge
date: Mon, 14 Oct 2024 19:27:01 GMT
```

We've a `join slack` in the footer: https://github.com/voxpupuli/voxpupuli.github.io/blob/master/_includes/footer.html#L8-L9

Reply:
ah yes. I should remove that too. Thanks